### PR TITLE
NonArduino: Tock: Fixup some bugs and expand the example

### PR DESCRIPTION
**Pull request by alistair23**

### CODE REVIEW

File: examples/NonArduino/Tock/libtockHal.h [hunk 1]
```diff
@@ -190,6 +190,10 @@ class TockHal : public RadioLibHal {
     void spiEnd() {
     }
 
+    void yield() {
+      yield();
+    }
+
```
Comment (jgromes) [inline examples/NonArduino/Tock/libtockHal.h:196]: Will this call a global yield or will it recursively call itself? If the intention is to call a global function, then I would suggest explicitly specifying it with `::yield();`.

File: examples/NonArduino/Tock/main.cpp [hunk 2]
```diff
@@ -30,9 +30,19 @@
 // include the hardware abstraction layer
 #include "libtockHal.h"
 
+// Include some libtock-c helpers
+#include <libtock/temperature.h>
+#include <libtock/humidity.h>
+
+#define BUFFER_LEN 64
+
```
Comment (jgromes) [inline examples/NonArduino/Tock/main.cpp:38]: I don't think it's a good idea to include specific sensors in the example. The example should be the necessary minimum. This pulls in more libraries and it also causes the functionality of the example to depend on the functionality of the sensors.

**Review by jgromes — COMMENTED** [review]
I added a couple comments which I think should be addressed.

File: examples/NonArduino/Tock/main.cpp (hunk 2 above)
Comment (alistair23) [reply examples/NonArduino/Tock/main.cpp:38]: I was thinking that it makes the example more useful for users. It gives more of an idea how the library can be used.

If you really don't think it should be there I can remove it. Would a receive example (useing #ifdef) be alright to add as though?

**Comment by alistair23** [comment]
Ok, I have dropped the sensor data

**Pull request merged**